Nos/Views/Note/NoteView.swift

```diff
@@ -61,7 +64,44 @@ struct NoteView: View {
     }
     
     var note: Event
-    
+
+    /// Retrieves a list of authors associated with the current note and sorts them 
+    /// by the number of mutual followees with the current author.
+    ///
+    /// - Returns: A sorted array of `Author` objects, in descending order of mutual followees with the current author.
+    private func getSortedAuthorsByMutualFollowees() -> [Author] {
```

Comment:
I think we can drop the "get" from the name.
```suggestion
    private func sortedAuthorsByMutualFollowees() -> [Author] {
```